### DISCUSSION TITLE
Support on source from PCI-E card like SDI baseband capture/playback card (Blackmagic SDI card) or ATSC/DVB demodulator

### DISCUSSION THREAD

**Discussion by un3creation** [ant-media/Ant-Media-Server#6890] [Ideas]
Hi there just wonder if there is any plan to support SDI capture/playback card and ATSC/DVB demodulator card. It would be welcoming addition for design the workflow

**Comment by filip-ams** [comment]
Greetings @un3creation,

Thank you for your question! Let me check with my team regarding our plans for the features you've mentioned.

In the meantime, could you share more details about your use case? Specifically:

How do you envision using SDI capture/playback or ATSC/DVB demodulator cards with Ant Media Server?

What kind of viewer count or scale are you targeting for your workflows?

This information will help us better understand your requirements and evaluate the opportunity from both your perspective and ours.

Looking forward to your input!

Best regards,
Filip
